build.cake

```diff
@@ -13,6 +12,9 @@ var UNIT_TEST_ASSEMBLY = "nunit-project-loader.tests.dll";
 
 var target = Argument("target", "Default");
 var configuration = Argument("configuration", "Debug");
+var nugetVersion = Argument("nugetVersion", (string)null);
+var chocoVersion = Argument("chocoVersion", (string)null);
+var binaries = Argument("binaries", (string)null);
```

Comment:
Download and unpack a NuGet package or whatever? Eventually, we could release from an Appveyor build or something. I think repackaging is rare enough a situation that it can require a little manual intervention.